### DISCUSSION TITLE
ZB-GW03 eWeLink Ethernet Zigbee Gateway based on an ESP32 and Silabs EFR32MG21 firmware hacked

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#819] [General]
FYI, ZB-GW03 eWeLink Ethernet Zigbee Gateway is based on an ESP32 and Silicon Labs EFR32 Zigbee and has been hacked with Tasmota.

https://templates.blakadder.com/ewelink_ZB-GW03

It looks to be an upgraded clone or spiritual successor to ITead Sonoff ZBBridge Zigbee Bridge and the hacking process for it is the same.

https://github.com/arendst/Tasmota/discussions/12764

![image](https://user-images.githubusercontent.com/6320001/134337396-2699f4fb-4339-4172-af98-4a47d4d5761b.png)

![image](https://user-images.githubusercontent.com/6320001/134337284-bd986202-e62e-4a61-8e0e-13d0d794d80c.png)

![image](https://user-images.githubusercontent.com/6320001/134337307-d98c7226-9f63-409d-8843-ba5e689ae749.png)

ZB-GW03 v1.0 and ZB-GW03-V1.2 is apparently sold rebranded under many names, including SmartWise and EACHEN brands:

https://www.okosabbotthon.hu/smartwise-zigbee-bridge-pro

https://ewelink.eachen.cc/product/eachen-ewelink-zigbee-bridge-pro-zbbridge-pro/

https://ewelinkcommunity.net/device-lists/zigbee/eachen-zb-gw03/

https://www.aliexpress.com/item/4000459723849.html

https://www.amazon.de/APP-Fernbedienung-Sprachsteuerung-Funktioniert-Verbindung-Smart-Produkten/dp/B094JKVLNR/

![image](https://user-images.githubusercontent.com/6320001/134337200-965ec994-aa6f-4ca0-a17b-c65082bcee5b.png)

![image](https://user-images.githubusercontent.com/6320001/134337244-c3d180cb-a999-4b5f-8ad2-74c629ccdc49.png)

Looks like is it is based on the same "SM-011 V1.0" EFR32MG21 module by CoolKit as ITead uses in their Sonoff ZBBridge Zigbee Bridge:

https://github.com/zigpy/zigpy/discussions/586

https://www.coolkit.cn/product/sm-011/

https://github.com/CoolKit-Technologies/DevDocs/tree/master/Zigbee

https://github.com/CoolKit-Technologies/DevDocs/blob/master/Zigbee/SM-011%E5%BA%94%E7%94%A8%E6%8C%87%E5%AF%BC%E4%B9%A6.md

https://translate.google.com/translate?hl=&sl=auto&tl=en&u=https%3A%2F%2Fgithub.com%2FCoolKit-Technologies%2FDevDocs%2Fblob%2Fmaster%2FZigbee%2FSM-011%E5%BA%94%E7%94%A8%E6%8C%87%E5%AF%BC%E4%B9%A6.md

**Comment by MattWestb** [comment]
Found it on Amazon for over 2 months ago https://github.com/zigpy/zigpy/discussions/635#discussioncomment-976568 and they still using the famous SM-011 module that was killing the original Sonoff ZBB.
But its one **IF** they have implanting WiFi coexistent with BT / Zigbee in the hard and software but i think the ESP32 cant do that.
